Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -23,7 +23,7 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
     weak var homePanelDelegate: HomePanelDelegate? = nil
     var source: BookmarksModel?
     var parentFolders = [BookmarkFolder]()
-    var bookmarkFolder = BookmarkRoots.MobileFolderGUID
+    var bookmarkFolder: BookmarkFolder?
```

Comment:
Humph. Maybe adjust visibility and signatures so you can do `self.source.folderForGUID(…)`? Or don't worry about it :)